### PR TITLE
Register akatsuki.is-a.dev

### PR DESCRIPTION
**Pull request by Miguel-altcell**
Register akatsuki.is-a.dev with CNAME record pointing to akatsuki-crew.rf.gd.

### CODE REVIEW

**Review by triach-rold — COMMENTED** [review]
403 forbidden error.

**Comment by Miguel-altcell** [comment]
done you can access it now.

File: domains/akatsuki.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Miguel-altcell",
+           "email": "wilsonmigueljalla@gmail.com",
+           "discord": "1223721772051075193"
+        },
+    
+        "record": {
+            "CNAME": "akatsuki-crew.rf.gd"
```
Comment (CuteDog5695) [inline domains/akatsuki.json:9]: website is accessible now, but I would like you to add more stuff to the website

**Review by iostpa — DISMISSED** [review]

File: domains/akatsuki.json (hunk 1 above)
Comment (Miguel-altcell) [reply domains/akatsuki.json:9]: Yeah im still fixing it. I just started and thank you (:

Comment (Miguel-altcell) [reply domains/akatsuki.json:9]: Hey im done now. And i know it looks bad but im still trying to get better

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by Miguel-altcell** [comment]
Hey i got an question.
I host my website at infinityfree.
What should i do now?  Or do i need to host on GitHub?

**Pull request merged**